### PR TITLE
Gantt design

### PR DESCRIPTION
**Pull request by MorsHs**
Added page redirection and inserted the chart.

naay notes sulod sa files changed na tab

**CableMap:**
Keep code:
line : **146**

**CableMapScript**
Keep code:
line:  87-99

**Code.js**
Keep code: doGet() function

Remove Code: 
757-760

Others: Keep

### CODE REVIEW

File: CableMap.html [hunk 1]
```diff
@@ -75,9 +76,15 @@
         <option value="TGN">TGN-IA/TGN-P</option>
       </select>
 
-      <div class="input-group-btn">
+      <div class="input-group-btn row">
```
Comment (MorsHs) [inline CableMap.html]: revert

File: CableMap.html [hunk 2]
```diff
@@ -75,9 +76,15 @@
...
         <button type="button" class="btn btn-primary" style="margin-right:15px;"
           onclick="searchCableSystem()">Search</button>
+
+        <div>
+          <div class="btn btn-primary" onclick="toTransferPage('gantt')">
```
Comment (MorsHs) [inline CableMap.html:84]: keep

File: CableMap.html [hunk 3]
```diff
@@ -75,9 +76,15 @@
...
+            gantt
+          </div>
+        </div>
```
Comment (MorsHs) [inline CableMap.html]: keep

File: CableMap.html [hunk 4]
```diff
@@ -91,7 +98,7 @@
 
 
   <div class="notif-panel" id="red-notif-panel">
-     <?!= HtmlService.createHtmlOutputFromFile('NotificationModal.html').getContent(); ?>
+    <?!= HtmlService.createHtmlOutputFromFile('NotificationModal.html').getContent(); ?>
```
Comment (MorsHs) [inline CableMap.html:112]: keep

File: CableMap.html [hunk 5]
```diff
@@ -135,8 +142,8 @@
 
 
 
-
-
+  <!-- for documentation purposes. It gets the current link of the page -->
+  <input type="hidden" id="url" value="<?= ScriptApp.getService().getUrl() ?>">
```
Comment (MorsHs) [inline CableMap.html:158]: keep

**Review by Crunchies3 — APPROVED** [review]

**Pull request merged**